src/lib/components/EmailLink/EmailLink.js

```diff
@@ -13,12 +13,12 @@ export const EmailLink = ({
   ...buttonUIProps
 }) => {
   const params = [];
-  if (bcc) params.push(`bcc=${bcc}`);
-  if (body) params.push(`body=${body}`);
-  if (cc) params.push(`cc=${cc}`);
-  if (subject) params.push(`subject=${subject}`);
+  if (bcc) params.push(`bcc=${escape(bcc)}`);
```

Comment:
`escape` is deprecated, see [here](https://stackoverflow.com/questions/26342123/replacement-for-javascript-escape) for replacement.
It would be great to add a unit test for this!